### PR TITLE
Fixes #13107: ZABBIX plugin

### PR DESCRIPTION
**Pull request by victorqrt**
https://www.rudder-project.org/redmine/issues/13107

### CODE REVIEW

File: zabbix/packaging/postinst [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+mkdir -p /var/rudder/plugin-resources
+echo "* */1 * * * /opt/rudder/bin/zabbix-plugin.py update " > /etc/cron.d/zabbix-rudder
```
Comment (peckpeck) [inline zabbix/packaging/postinst]: Same remark as in centreon

**Comment by VictorQrt** [comment]
Commit modified

**Comment by VictorQrt** [comment]
Commit modified

**Comment by peckpeck** [comment]
OK, merging this PR

**Pull request merged**